command.go

```diff
@@ -156,5 +166,8 @@ func (c Command) startApp(ctx *Context) error {
 		app.Action = helpSubcommand.Action
 	}
 
+	// set the writer to the original App's writer
```

Comment:
This is a separate bug that I noticed when I tried to integrate the latest code with our codebase. The `Writer` wasn't copied to the sub command, so subcommands got a new `os.Stdout` `Writer` even though I set the `App`'s `Writer` to something else to capture output in my tests.